packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -178,8 +181,31 @@ const requestRemoteNode = (url, headers, tmpFilename, httpOpts, attempt = 1) =>
 
     responseStream.on(`response`, response => {
       resetTimeout()
+      contentLength =
+        response.headers && Number(response.headers[`content-length`])
```

Comment:
is `content-length` required header? If it's not set then `Number(undefined)` will be `NaN` and then we will faill all that do not provide it

Reply:
ok - scratch above / this is actually handled

because if `contentLength` would be `NaN` and `NaN` is actually falsy it seems

Reply:
Yes, exactly

Reply:
I did ask the same question myself but tested it :joy: